code/04-drawing-bitwise/MaskAnd.py

```diff
@@ -2,24 +2,21 @@
  * Python program to apply a mask to an image.
```

Comment:
I'm not sure of the terminology here, but should "program" be replaced with "code" here as it is elsewhere?

Reply:
ah, the _Python code_ code thing is a remainder of the notebook idea, where I wouldn't want to say _program_. I will revert all those, since we're doing scripts now.